extensions/guacamole-auth-cas/src/main/java/org/apache/guacamole/auth/cas/ticket/TicketValidationService.java

```diff
@@ -33,10 +33,14 @@
 import java.util.HashMap;
 import java.util.Map;
 import java.util.Map.Entry;
+import java.util.Set;
+import java.util.HashSet;
+import java.util.regex.*;
```

Comment:
We generally try to avoid importing classes using wildcards like this, and prefer to list out the specific ones you need.